### DISCUSSION TITLE
Github Education: We need more information demonstrating your academic affiliation.

### DISCUSSION THREAD

**Discussion by S0lrdSd** [community/community#51138] [GitHub Education]
Github rejects my applications within five minutes, even though I provide a letter stating my affiliation with an academic institution (the original is in non-English, but there is a paper with a translation next to it). I have tried indicating. Probably rejected because of the proof type, I chose Dated enrollment letter on school letterhead and Other (A letter indicating my affiliation with an academic institution).
This letter has the date, but Github still rejects applications

![image](https://user-images.githubusercontent.com/128751229/227990192-1b2c264e-b5fa-4660-af7b-6598a982d2cd.png)

**Comment by S0lrdSd** [comment]
I tried to send a screenshot of the school portal, the request was also rejected

**Comment by samus-aran** [comment]
Hi @jurmraw 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. 

Hope this helps 👍

**Reply by S0lrdSd** [reply]
Thank you for your answer!
The first and last names are the same as in my school letter. I tried to take pictures on my phone camera (directly to the site github), attach a photo from the computer and take a photo from a webcam. In none of these cases did Github approve my application, although the translation of the letter is legible.
Maybe it’s because my school doesn’t have domain mail. I’ll try to talk to my computer science teacher, but I don’t think that’s gonna change that.
